src/react-aria-modal.js

```diff
@@ -193,7 +185,7 @@ class Modal extends React.Component {
 
     if (props.dialogStyle) {
       for (const key in props.dialogStyle) {
-        if (!props.dialogStyle.hasOwnProperty(key)) continue;
+        if (!props.dialogStyle.includes(key)) continue;
```

Comment:
AFAIK `underlayStyle` is an object, and does not have the `includes` prototype method. Could you explain this a little bit more?

Reply:
Same as above... however eslint is unhappy with `hasOwnProperty`.

Reply:
> Same as above... however eslint is unhappy with `hasOwnProperty`.

@resource11

This is a security issue. You can use a function like the following to "make eslint happy".

```javascript
const hasOwnProperty = (object, prop) => Object
  .prototype
  .hasOwnProperty
  .call(object, prop)
```

Reply:
Thanks, @rivajunior!